src/editor/base-editor.js

```
@@ -68,7 +68,9 @@ define( [ "core/eventmanager", "util/scrollbars", "ui/widget/tooltip" ],
      */
     extendObject.close = function() {
       // Remove the editor's root element from the element to which it was attached
-      extendObject.rootElement.parentNode.removeChild( extendObject.rootElement );
+      if ( extendObject.rootElement.parentNode ) {
```

With the change you made here https://github.com/mozilla/butter/pull/938/files#L1R81, this check might be unnecessary. I'd rather have this function fail as an obvious sign that we call `close` way too much than it half-fail silently.